aws/resource_aws_secretsmanager_secret.go

```diff
@@ -77,17 +77,22 @@ func resourceAwsSecretsManagerSecret() *schema.Resource {
 				},
 			},
 			"rotation_enabled": {
-				Type:     schema.TypeBool,
-				Computed: true,
+				Deprecated: "Use the aws_secretsmanager_secret_rotation resource instead",
+				Type:       schema.TypeBool,
+				Computed:   true,
 			},
 			"rotation_lambda_arn": {
```

Comment:
Since these attributes are `Computed: true` now (which is required for supporting the new resource), it will no longer be possible to remove this configuration and have Terraform remove the rotation in _this_ resource. I will add some additional verbiage in the resource documentation and CHANGELOG around this behavior change.